file.js

```diff
@@ -14,3 +16,83 @@ document.querySelectorAll('.link').forEach((n) => n.addEventListener('click', ()
   navbar.classList.remove('active');
   container.classList.remove('active');
 }));
+
+// popup window
+const myProjects = [
```

Comment:
- [ ] Following the previous request, your implementation did not fix the issue as required. Actually, your popup design is still dissimilar to the one provided in the Figma template:

1. On some mobile screen sizes, the close button disappears, kindly fix that.
2. Your popup content on mobile doesn't have any padding. Please make sure it matches as possible the Figma template(padding, margin, background, font...).

- Images reference:

**Your popup:**
![Screenshot from 2022-07-22 19-52-04](https://user-images.githubusercontent.com/80612925/180497446-ef3d7c6f-5d05-4959-a3cd-5c8c8c8adb74.png)

**Figma popup:**
![image](https://user-images.githubusercontent.com/80612925/180497483-4dcfdbfd-e7da-438a-a448-55736a77d89a.png)